specify current limitation (#109078)
Azure/azure-content: articles/active-directory/develop/scenario-web-api-call-api-app-configuration.md

### Option 1: Call Microsoft Graph

To call Microsoft Graph, *Microsoft.Identity.Web* enables you to directly use the `GraphServiceClient` (exposed by the Microsoft Graph SDK) in the API actions. 

> >[!NOTE] 

Reviewer: ```suggestion

>[!NOTE] 
```

> >[!NOTE] 

Reviewer: ```suggestion

>[!NOTE] 
```
Apologies, an additional space is needed!

### Option 1: Call Microsoft Graph

To call Microsoft Graph, *Microsoft.Identity.Web* enables you to directly use the `GraphServiceClient` (exposed by the Microsoft Graph SDK) in the API actions. 

>[!NOTE] 

> > There is an ongoing issue for Microsoft Graph SDK v5+, for more information, refer to the [Github issue](https://github.com/AzureAD/microsoft-identity-web/issues/2097)

Reviewer 2: ```suggestion
> There is an ongoing issue for Microsoft Graph SDK v5+. For more information, refer to the [GitHub issue](https://github.com/AzureAD/microsoft-identity-web/issues/2097).
```

### Option 1: Call Microsoft Graph

> To call Microsoft Graph, *Microsoft.Identity.Web* enables you to directly use the `GraphServiceClient` (exposed by the Microsoft Graph SDK) in the API actions (this currently does not work for Microsoft Graph SDK v5+, see https://github.com/AzureAD/microsoft-identity-web/issues/2097). To expose Microsoft Graph:

Reviewer: ```suggestion
To call Microsoft Graph, *Microsoft.Identity.Web* enables you to directly use the `GraphServiceClient` (exposed by the Microsoft Graph SDK) in the API actions. 
>[!NOTE] 
> There is an ongoing issue for Microsoft Graph SDK v5+, for more information, refer to the [Github issue](https://github.com/AzureAD/microsoft-identity-web/issues/2097)

To expose Microsoft Graph:
```

Reviewer: Just change it to the following! Thanks
